tests/x11/kontact.pm

```diff
@@ -22,9 +22,19 @@ sub run() {
     # To disable it run at first time start
     x11_start_program("echo \"[General]\" >> ~/.kde4/share/config/kmail2rc");
     x11_start_program("echo \"first-start=false\" >> ~/.kde4/share/config/kmail2rc");
-    sleep 2;
+
+    # this is the workaround as same as above but against kontact5
+    # once we got all KF5-based kdepim merged, we can drop the KDE4 workaround
+    x11_start_program("echo \"[General]\" >> ~/.config/kmail2rc");
```

Comment:
please see the comment https://github.com/nilxam/os-autoinst-distri-opensuse/blob/fix_kde_15121/tests/x11/kontact.pm#L21 it's not a bug in kontact even not a real workaround for kontact, *but* a workaround for this test ie. for openqa, in the past we have a complicate logic in this test against this annoying behavior till we're decided to disable the first-run check in its config file.